src/RuleSet/RuleSets.php

```diff
@@ -10,22 +10,130 @@
  * with this source code in the file LICENSE.
  */
 
-namespace PhpCsFixer;
+namespace PhpCsFixer\RuleSet;
 
 use PhpCsFixer\Fixer\FunctionNotation\NativeFunctionInvocationFixer;
 use PhpCsFixer\Fixer\PhpUnit\PhpUnitTargetVersion;
 
 /**
- * Set of rules to be used by fixer.
+ * Set of rule sets to be used by fixer.
  *
- * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  * @author SpacePossum
  *
  * @internal
  */
-final class RuleSet implements RuleSetInterface
+final class RuleSets
```

Comment:
question: why dropping the interface ?

Reply:
it is not dropped, you can see it in use by the `RuleSet` class

Reply:
ah, it's different class, cool!